mlflow/utils/search_utils.py

```diff
@@ -104,14 +86,12 @@ def _get_identifier(cls, identifier):
         try:
             entity_type, key = identifier.split(".", 1)
         except ValueError:
-            raise MlflowException("Invalid filter string '%s'. Filter comparison is expected as "
-                                  "'attribute.<key> <comparator> <value>', "
-                                  "'metric.<key> <comparator> <value>', "
-                                  "'tag.<key> <comparator> <value>', or "
-                                  "'params.<key> <comparator> <value>'." % identifier,
+            raise MlflowException("Invalid identifier '%s'. Columns should be specified as "
```

Comment:
Made this text a bit more general so that we can reuse the helper in the parse_order_by.